trl/trainer/ppo_config.py

```diff
@@ -41,199 +28,35 @@ class PPOConfig:
     command line.
 
     Parameters:
-        exp_name (`str`, *optional*, defaults to `os.path.basename(__file__)[: -len(".py")]`):
+        exp_name (`str`, *optional*, defaults to `os.path.basename(__file__)[:-3]`):
```

Comment:
I've always been a bit unsure why we need this extra parameter in our configs. Should we remove it (maybe in a follow-up PR)?